base_search_mail_content/README.rst

```diff
@@ -3,43 +3,45 @@
    :alt: License: AGPL-3
 
 ========================
-Mail message name search
+Base Search Mail Content
 ========================
 
-This module adds the capability to search for mail messages by subject or
-body of the message. This will be useful in models that make intense use of
-messages, like project issues or helpdesk tickets.
+This module adds the capability to find any object (e.g. project issues or
```

Comment:
Should be : This module adds the capability to find **on** any object (e.g. project issues or